providers/jcr_node.rb

```diff
@@ -81,12 +77,13 @@ def make_url(new_resource)
   url = make_url(new_resource)
   if check_node(url, new_resource.user, new_resource.password, new_resource.name)
     # If the node exists, delete it
-    fields = [Curl::PostField.content(':operation', 'delete')]
-    c = curl_form(url, new_resource.user, new_resource.password, fields)
-    if c.response_code == 200 || c.response_code == 201
+    # EH might not have to be multipart
```

Comment:
also this comment might confuse more than help, it was a TODO for myself, so, just throw it away :)